samples/tfm_integration/psa_firmware/README.rst

```diff
@@ -17,52 +17,71 @@ It uses **IPC Mode** for communication, where an IPC mechanism is inserted to
 handle secure TF-M API calls and responses. The OS-specific code to handle
 the IPC calls is in ``tfm_ipc.c``.
 
+TF-M supports three types of firmware upgrade mechanisms:
+``https://tf-m-user-guide.trustedfirmware.org/docs/technical_references/design_docs/tfm_secure_boot.html#firmware-upgrade-operation``
+
+This example uses the overwrite firmware upgrade mechanism, in particular, it showcases
+upgrading the non-secure image which can be built from any other sample in the
+``zephyr/samples/tfm_integration`` directory.
+
 The sample prints test info to the console either as a single-thread or
 multi-thread application.
 
+
 Building and Running
 ********************
 
 This project needs another firmware as the update payload. It must use another
 example's hex file, and should be specified on the command line
-as ZEPHYR_FIRMWARE_UPDATE_SAMPLE_BUILD. For example, to use the sample
-`tfm_integration/tfm_ipc` as the payload:
-
-.. code-block:: bash
-
-   cd <ZEPHYR_ROOT>
-   west build -p -b lpcxpresso5s69_ns samples/tfm_integration/psa_firmware \
-       -d build/lpcxpresso55s69_ns/tfm_integration/psa_firmware
-       -- -DCONFIG_FIRMWARE_UPDATE_IMAGE=`realpath build/lpcxpresso55s69_ns/tfm_integration/tfm_ipc/zephyr_ns_signed.hex`
+as ``CONFIG_APP_FIRMWARE_UPDATE_IMAGE``.
 
-
-This project outputs startup status and info to the console. It can be built and
-executed on an ARM Cortex M33 target board or QEMU.
+To use the ``tfm_integration/tfm_ipc`` sample as the NS firmware update
+payload, follow the instructions below:
 
 This sample will only build on a Linux or macOS development system
 (not Windows), and has been tested on the following setups:
```

Comment:
Not any particular reason as I don't have a setup for windows. I confirmed that Jimmy tested on mac and I tested on Ubuntu.

Reply:
I think this may have been something with very early versions of TF-M (I've since forgotten why!) that no longer applies. I don't have a Windows machine to test on, but should probably setup a VM at some point. We can cycle back to this in another PR once I can confirm everything works in Windows, though. It's a good idea to check anyway, and make sure the rest of the documentation is out to date, but out of scope here.